File: Ambilight/ShooterGDI/Sector.cpp
#include "stdafx.h"
#include "Sector.h"

namespace ScreenshotLogicCPP {

	Sector::Sector(int x, int y, int width, int height, int newIndex)
	{
		Top = y;
		Left = x;
		Bottom = y + height;
		Right = x + width;

		index = newIndex;
	}

}

File: Ambilight/Ambilight/Helpers/Sector.cs
namespace AmadeusW.Ambilight.Helpers
{
    public class Sector
    {
        public enum ScreenEdge
        {
            Top,
            Right,
            Bottom,
            Left
        };

        public System.Drawing.Rectangle Area;
        public int Index;
        public bool SpecialMiddleSector;
        public int Anchor;
        public ScreenEdge Edge;
    }
}
